flink-staging/flink-hbase/pom.xml

```diff
@@ -112,6 +112,12 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
```

Comment:
Unfortunately the TableInputFormat and TableOutputFormat are in the server jar.
For the read we've reimplemented it to make it more robust so we don't need that jar, but for the output it is indeed required.